### PR TITLE
Removed tailwind from editor

### LINKED ISSUE #72: Remove tailwind from package
We are using tailwind just to style the slash menu and a few other things. We could use custom SCSS there and get rid of tailwindCSS from the package. 

enhancement _t

### PR DESCRIPTION
**Pull request by goutham-subramanyam**
Fixes #72 

@karthiknmenon _a Please review.

cc: @amaldinesh7 

Note: Tailwind is still being used for the catalog.

### CODE REVIEW

**Comment by karthiknmenon** [comment]
@goutham-subramanyam _a Two minor changes. 
1. Can we clean up the styles used to show variables used in editor?
image
2. And, can we do the same for mentions as well? 
image

Please assign it to Amal for final review. 

cc: @amaldinesh7

**Comment by goutham-subramanyam** [comment]
@karthiknmenon Updated. Please check:

Screenshot 2021-12-15 at 6 56 32 PM

Screenshot 2021-12-15 at 6 56 28 PM

@amaldinesh7 _a Please review.

File: lib/styles/editor/_editor-content.scss [hunk 1]
```diff
@@ -64,14 +46,12 @@ $list-item-color: #1a141f;
   // Blockquote
   blockquote {
     font-weight: $neeto-ui-font-medium;
-    font-style: italic;
```
Comment (amaldinesh7) [inline lib/styles/editor/_editor-content.scss:67]: We need italics for the blockquote item right?

Previously
Screenshot 2021-12-16 at 12 40 20 PM

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@goutham-subramanyam _a I've added one comment. Please have a look.

File: lib/styles/editor/_editor-content.scss (hunk 1 above)
Comment (goutham-subramanyam) [reply lib/styles/editor/_editor-content.scss:67]: It was removed purposely. Can be reverted if needed.

**Comment by amaldinesh7** [comment]
@goutham-subramanyam _a Changes looks good to me. Please resolve the conflict and merge it.

**Pull request merged**